packages/flutter/test/widgets/focus_manager_test.dart

```diff
@@ -387,8 +391,12 @@ void main() {
     });
 
     testWidgets('FocusManager responds to app lifecycle changes.', (WidgetTester tester) async {
-      final bool shouldRespond = kIsWeb || defaultTargetPlatform != TargetPlatform.android;
```

Comment:
Similar question with the above test, can we just rely on the test variants here?